.github/workflows/conda-build.yml

```diff
@@ -22,13 +22,13 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
-    - name: Setup python 3.7
+    - name: Setup python 3.8
       uses: actions/setup-python@v2
       with:
-        python-version: 3.7
+        python-version: 3.8
     - name: Pull codebuild docker image
       run: |
         docker pull public.ecr.aws/codebuild/amazonlinux2-x86_64-standard:3.0
     - name: Build conda environment
       run: |
-        ./nbi/codebuild_build.sh -i public.ecr.aws/codebuild/amazonlinux2-x86_64-standard:3.0 -a envs -b nbi/buildspec.yml
\ No newline at end of file
+        ./nbi/codebuild_build.sh -i public.ecr.aws/codebuild/amazonlinux2-x86_64-standard:3.0 -a envs -b nbi/buildspec.yml
```

Comment:
What changed on this line?

Reply:
The red "no entry" circle means there's no newline at the end of the file; the GitHub editor automatically adds a newline.